app/controllers/servers_controller.rb

```diff
@@ -0,0 +1,156 @@
+  #
+  # Expected params:
+  # {
+  #   "server": {
+  #     "url": String,                 # Required: URL of the BigBlueButton server
+  #     "secret": String,              # Required: Secret key of the BigBlueButton server
+  #     "load_multiplier": Float       # Optional: A non-zero number, defaults to 1.0 if not provided or zero
+  #   }
+  # }
+  def create
+    if server_create_params[:url].blank? || server_create_params[:secret].blank?
+      render json: { message: 'Error: Please input at least a URL and a secret!' }, status: :bad_request
+    else
+      tmp_load_multiplier = server_create_params[:load_multiplier].presence&.to_d || 1.0
+      tmp_load_multiplier = 1.0 if tmp_load_multiplier.zero?
+
+      server = Server.create!(url: server_create_params[:url], secret: server_create_params[:secret], load_multiplier: tmp_load_multiplier)
+      render json: { id: server.id }, status: :created
+    end
+  end
+
+  # Update a BigBlueButton server
+  # PUT /bigbluebutton/api/v1/servers/:id
+  #
+  # Expected params:
+  # {
+  #   "server": {
+  #     "state": String,         # Optional: 'enable', 'cordon', or 'disable'
+  #     "load_multiplier": Float # Optional: A non-zero number
+  #     "secret": String         # Optional: Secret key of the BigBlueButton server
+  #   }
+  # }
+  def update
+    begin
+      updated_server = ServerUpdateService.new(@server, server_update_params).call
+      render json: { server: updated_server }, status: :ok
+    rescue StandardError => e
+      render json: { error: e.message }, status: :bad_request
+    end
+  end
+
+  # Remove a BigBlueButton server
+  # DELETE /bigbluebutton/api/v1/servers/:id
+  def destroy
+    begin
+      @server.destroy!
+      render json: { id: @server.id }, status: :ok
+    rescue ApplicationRedisRecord::RecordNotDestroyed => e
+      render json: { error: e.message }, status: :bad_request
+    end
+  end
+
+  # Set a BigBlueButton server as unavailable and clear all meetings from it
+  # POST /bigbluebutton/api/v1/servers/:id/panic
+  #
+  # Expected params:
+  # {
+  #   "server": {
+  #     "keep_state": Boolean # Optional: Set to 'true' if you want to keep the server's state after panicking, defaults to 'false'
+  #   }
+  # }
+  def panic
+    begin
+      keep_state = (server_panic_params[:keep_state].presence || false)
+      meetings = Meeting.all.select { |m| m.server_id == @server.id }
+      meetings.each do |meeting|
+        Rails.logger.debug { "Clearing Meeting id=#{meeting.id}" }
+        moderator_pw = meeting.try(:moderator_pw)
+        meeting.destroy!
+        get_post_req(encode_bbb_uri('end', @server.url, @server.secret, meetingID: meeting.id, password: moderator_pw))
+      rescue StandardError => e
+        render json: { error: "Could not destroy meeting id=#{meeting.id}: #{e}" }, status: :bad_request
+      end
+
+      @server.state = 'disabled' unless keep_state
+      if @server.save!
+        render json: { server: @server }, status: :ok
```

Comment:
Returning the whole server here seems weird to me. Maybe just the id with a success mesage?